tests/Basketball/Coach.java

```diff
@@ -0,0 +1,8 @@
+package Basketball;
+
+
+public class Coach extends Human {
```

Comment:
No need. Coach doesn't have anything extra that a human would not. Coach should simply be anew instance of a human